### PR TITLE
Solved Issue #8077

### PR DESCRIPTION
**Pull request by Ajinkyap331**
<hr>

## Description 📜

Solved Issue #8077 as a Source of Documentation
Solved Issue #8077 

<hr>

## Type of change 📝

- [x] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev #React/Angular/Vue

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Comment by Ajinkyap331** [comment]
@anandhu720 @heet-kumar please review this

**Comment by anandhu720** [comment]
@heet-kumar please review it

**Comment by heet-kumar** [comment]
> @heet-kumar please review it

sure bro

File: Frontend_Web_Development_React_Angular_Vue/React/Week 8/Unfetch.md
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 8/Unfetch.md:53]: just add explanation and process of these code.

File: Frontend_Web_Development_React_Angular_Vue/React/Week 8/Unfetch.md
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 8/Unfetch.md:104]: here also just add explanation and process of these code.

**Review by heet-kumar — CHANGES_REQUESTED** [review]
@Ajinkyap331 add explanation and process of the code

**Comment by Ajinkyap331** [comment]
Ok I will do it today itself @heet-kumar

**Comment by Ajinkyap331** [comment]
@heet-kumar  review this Please

**Review by heet-kumar — CHANGES_REQUESTED** [review]
@Ajinkyap331 add theory explanation in short I mean add more content because theory explanation/content is less so add that also.

**Comment by Ajinkyap331** [comment]
@heet-kumar Can I do it tomorrow i.e 26th

**Comment by heet-kumar** [comment]
@Ajinkyap331 ok no problem try to do asap.

**Comment by Ajinkyap331** [comment]
@heet-kumar I have done all I can do cause only documentation is the place where I can get the info NO tutorial NO youtube video. I guess I have added many things

**Comment by heet-kumar** [comment]
@Ajinkyap331 I only want that, just add how these code is working which you have added in the documentation, just explain the process of the code just below that code.

**Comment by Ajinkyap331** [comment]
@heet-kumar thanks I will do it now itself

**Comment by heet-kumar** [comment]
> @heet-kumar thanks I will do it now itself

ok do it then we will merge it PR.

**Comment by Ajinkyap331** [comment]
@heet-kumar  Done

**Review by heet-kumar — APPROVED** [review]
@Ajinkyap331 Nice explanation LGTM PR !! 👍 let's wait for the supervisor approval if they mention any changes then make that to.

**Comment by heet-kumar** [comment]
@anandhu720 once review from your side get merged this PR.

**Comment by Ajinkyap331** [comment]
> @Ajinkyap331 Nice explanation LGTM PR !! 👍 let's wait for the supervisor approval if they mention any changes then make that to.

LGTM meaning?

**Comment by heet-kumar** [comment]
> > @Ajinkyap331 Nice explanation LGTM PR !! 👍 let's wait for the supervisor approval if they mention any changes then make that to.
> 
> LGTM meaning?

Lets get the merge PR

**Comment by Ajinkyap331** [comment]
> > > @Ajinkyap331 Nice explanation LGTM PR !! 👍 let's wait for the supervisor approval if they mention any changes then make that to.
> > 
> > 
> > LGTM meaning?
> 
> Lets get the merge PR

OK

**Review by anandhu720 — APPROVED** [review]

**Pull request merged**
